### PR TITLE
Documentation in Markdown for Depends Dir

### PR DESCRIPTION
**Pull request by super3**
Documentation more readable when viewed on Github.

### CODE REVIEW

File: depends/README_USAGE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+To build dependencies for the current arch+OS:
+
+    make
+
+To build for another arch/OS:
+
+	make HOST=host-platform-triplet && make HOST=host-platform-triplet
```
Comment (laanwj) [inline depends/README_USAGE.md]: btw this double make makes no sense, what as the intention here @cfields?

Comment (theuni) [inline depends/README_USAGE.md]: whoops. This is just busted c/p from an initial version of depends that worked differently (required make && make install). Wrong on several levels :)

The second part can just be dropped.

Comment (super3) [inline depends/README_USAGE.md]: So just change to:

     To build for another arch/OS:
          make HOST=host-platform-triplet

Comment (laanwj) [inline depends/README_USAGE.md]: @super3: yes

**Comment by super3** [comment]
@laanwj @theuni Should be all fixed now.

**Comment by laanwj** [comment]
Merged via 16e8ea3 (with some extra improvements)

**Pull request closed without merging**